packages/provider-elasticsearch/src/example-types/results/type.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Typings for the result example type.
+ */
+
+interface HighlightConfig {
+  /**
+   * Whether to send highlighting configuration to elastic and merge
+   * highlighting results onto source. Defaults to `false`.
+   */
+  enable?: boolean
```

Comment:
I think the default behavior should be to highlight, so we should invert this to a `disable` flag

Reply:
Highlighting is a perf hit so why would that be the default?

Reply:
Based on our conversation, we should default this on. The performance concern would be for searches not powering a UI, but there's precedent for thinking about this with flags like `forceFilterOnly`.

Some API cases:

`highlight: {disable: true }` disables highlighting
`highlight: undefined` (or missing) would highlight
`highlight: { /*... config ...*/ }` would highlight with config